moderators.md

```diff
@@ -0,0 +1,143 @@
+---
+title: "Moderators"
+---
+
+A [moderator](governance.md#moderator) is a volunteer gatekeeper
```

Comment:
I think you can try running aeolus for better linebreaks: https://docs.ropensci.org/aeolus/

Reply:
Used `aeolus` for 1b92d68c94435a2dcba75f6c94dcaeab80ad4744